packages/link/src/Link.ts

```diff
@@ -102,6 +102,16 @@ function eventsListener(
           iframeUrlObject?.origin || 'https://web.meshconnect.com'
         )
       }
+      if (currentOptions?.injectedCoreConnectors) {
+        //or use currentOptions?.injectedConnectors
+        iframeElement().contentWindow?.postMessage(
```

Comment:
Let's introduce a helper function for sending messages to the Link UI, we will use it a lot.